### PR TITLE
feat: update logging operator to the latest stable version test it with k8s 1.25

### PR DESCRIPTION
**Pull request by nutellinoit**
e2e is failing on opensearch, I already tested logging operator on a 1.25 cluster

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -564,6 +462,7 @@ trigger:
     include:
       - refs/tags/**
 
+load:
```
Comment (ralgozino) [inline .drone.yml:465]: is this OK?

**Review by ralgozino — APPROVED** [review]
minor check on a change in drone.yaml
the rest lgtm

File: .drone.yml (hunk 1 above)
Comment (nutellinoit) [reply .drone.yml:465]: Let me check

Comment (nutellinoit) [reply .drone.yml]: I don't know why it was there, fix'd

**Review by ralgozino — APPROVED** [review]
🚀

**Pull request merged**